projects/canopy/src/lib/carousel/carousel.notes.ts

```diff
@@ -39,4 +39,6 @@ and in your HTML:
 | \`\`slideDuration\`\` | Duration in milliseconds of the transition between slides.                                               | number  |   500     |   No     |
 | \`\`description\`\`   | Text used to describe the carousel content for screen readers. This is visually hidden.                  | string  | undefined |   Yes    |
 | \`\`headingLevel\`\`  | The heading level of the description: \`\`1\`\`, \`\`2\`\`, \`\`3\`\`, \`\`4\`\`, \`\`5\`\`, \`\`6\`\`   | number  | undefined |   Yes    |
+| \`\`autoPlay\`\`      | Enables auto play mode when set to true.                                                                 | boolean |   false   |   No     |
+| \`\`autoPlayDelay\`\` | Delay time in miliseconds to switch to next slide when autoPlay mode is On.                              | number  |   2000    |   No     |
```

Comment:
I guess because the `autoPlay` input is a boolean I would say `Delay time in milliseconds to switch to next slide when autoPlay mode is set to true.`?
There is also a tiny typo in `miliseconds` -> `milliseconds `